rules/CodeQuality/Rector/FuncCall/SortNamedParamRector.php

```diff
@@ -141,7 +141,10 @@ static function (Arg $arg1, Arg $arg2) use ($order): int {
                 /** @var Identifier $argName2 */
                 $argName2 = $arg2->name;
 
-                return $order[$argName1->name] <=> $order[$argName2->name];
+                $order1 = $order[$argName1->name] ?? PHP_INT_MAX;
+                $order2 = $order[$argName2->name] ?? PHP_INT_MAX;
```

Comment:
how about use `?? null` instead?

```suggestion
                $order1 = $order[$argName1->name] ?? null;
                $order2 = $order[$argName2->name] ?? null;
```

Reply:
No, the max int was intentional so that the argument that don't exist are moved towards the back